### PR TITLE
Radius spelling standardization

### PR DESCRIPTION
**Pull request by jasonviviano**
Thank you for helping make the Radius documentation better!
 
**Please follow this checklist before submitting:**
 
- [ ] [Read the contribution guide](https://docs.radapp.dev/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs
 
In addition, please fill out the following to help reviewers understand this pull request:
 
## Description
 
Too many times have we suffered from unknowns regarding whether Radius Application, Radius application, Radius Environment, Radius environment, Recipe, recipe or anything else is correct. We can standardize it and just put it in the contributor section of our docs as reference.
 
Screenshot 2023-04-20 at 11 57 59 AM

## Issue reference
 
https://github.com/project-radius/docs/issues/540

### CODE REVIEW

File: docs/content/contributing/contributing-docs.md [hunk 1]
```diff
@@ -100,8 +100,19 @@ These conventions should be followed throughout all Radius documentation to ensu
...
 The docs pipeline uses [aspell](http://aspell.net/) to check for spelling mistakes. If you need to add a new custom word to the allow-list, update `.github/config/en-custom.txt`.
 
+### Radius concepts
+
+This convention of capitalization should be followed throughout all Radius documentation to ensure a consistent experience across all docs when introducing Radius specific concepts.
+
+- **Radius Environment**
+- **Radius Application**
+- **Radius Recipe**
+- **Radius Link**
```
Comment (willtsai) [inline docs/content/contributing/contributing-docs.md]: make a note that these are examples so that readers don't think it's a comprehensive list

Comment (jasonviviano) [reply docs/content/contributing/contributing-docs.md]: Updated it! Followed the Dapr style you showed me earlier

File: docs/content/contributing/contributing-docs.md [hunk 2]
```diff
@@ -91,12 +91,14 @@ Get up and running with a GitHub Codespace in seconds. This will give you a full
...
-- **Headers and titles** - Headers and titles must be descriptive and clear, use sentence casing i.e. use the above casing guidance for headers and titles too
-- **Use simple sentences** - Easy-to-read sentences mean the reader can quickly use the guidance you share.
-- **Avoid the first person** - Use second person "you", "your" instead of "I", "we", "our".
-- **Assume a new developer audience** - Some obvious steps can seem hard. E.g. Now set an environment variable Radius to a value X. It is better to give the reader the explicit command to do this, rather than having them figure this out.
-- **Use present tense** - Avoid sentences like "this command will install redis", which implies the action is in the future. Instead use "This command installs redis" which is in the present tense.
+| Style/Tone | Guidance |
+|---------|-------------|
+|**Casing**|Use upper case only:<ul><li>At the start of a sentence or header</li><li>For proper nouns including names of technologies (Redis, Kubernetes, etc.)</li> <li>Radius conventions (Radius Recipe, Radius Environment, Radius Application, etc.)</li></ul>|
```
Comment (willtsai) [inline docs/content/contributing/contributing-docs.md]:
```suggestion
|**Casing**|Use upper case only:<ul><li>At the start of a sentence or header</li><li>For proper nouns including names of technologies (Redis, Kubernetes, etc.)</li> <li>Names of Radius concepts (Radius Recipe, Radius Environment, Radius Application, etc.)</li></ul>|
```

**Review by willtsai — APPROVED** [review]

**Review by AaronCrawfis — APPROVED** [review]
Table looks good! Could you also make sure there's an issue tracking us updating our existing docs to follow this guidance?

Thanks!

**Comment by jasonviviano** [comment]
> Table looks good! Could you also make sure there's an issue tracking us updating our existing docs to follow this guidance?
> 
> Thanks!

Done! https://github.com/project-radius/docs/issues/550

**Pull request merged**